### ISSUE TITLE
Menu Bar Rendering Wrong

### ISSUE THREAD

**Issue by zm711** [SpikeInterface/spikeinterface-gui#40]
@samuelgarcia 
Howdy Samuel, 
While testing the installation_tips from spikeinterface for windows and Mac, both had a rendering issue with the menus for the gui:
I'm happy to try to fix this, but to be honest I don't know much about Qt stuff. I saw in your code that you were thinking about setting stylesheets in `mainwindow.py` so maybe that would help and if you're working on other stuff I could play around with that.

I've attached the rendering issue below.

![image](https://user-images.githubusercontent.com/92116279/223134513-75331ec4-c45f-436d-b6c6-d84b6d842340.png)

**Comment by samuelgarcia** [comment]
Hi Zach, 
you are aexploring all spikeinertface zone!!! thank you.

Having button in dock bar is a bit hackish in QT.
In sigui, this happen here https://github.com/SpikeInterface/spikeinterface-gui/blob/main/spikeinterface_gui/mainwindow.py#L114

I guess that we have to play with the bar/button size or reduce the police size but I have to admit that I did not fitgh too much on this.
If you have time, this is nice but do not spend too much on this!

**Comment by alejoe91** [comment]
@zm711 is this fixed?

**Comment by zm711** [comment]
No, it has to do with high res screens (like on mac laptops). We need to come up with a hack that determines higher res vs lower res screens to handle this, but since I don't know qt at all I haven't really tried messing with this too much.
